src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -122,6 +124,8 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     case UpdateWorkspace(workspaceName, operations) => pipe(updateWorkspace(workspaceName, operations)) to sender
     case ListWorkspaces => pipe(listWorkspaces()) to sender
     case ListAllWorkspaces => pipe(listAllWorkspaces()) to sender
+    case ListPublishedWorkspaces => pipe(listPublishedWorkspaces()) to sender
+    case ListWorkspacesWithAttribute(attributeMap) => pipe(listWorkspacesWithAttribute(attributeMap)) to sender
```

Comment:
The convention is to prefix admin functions with Admin (e.g. AdminListPublishedWorkspaces). Also the asFCAdmin should be up here:
case AdminListPublishedWorkspaces => pipe(asFCAdmin { listPublishedWorkspaces() }) to sender